content/pt-br/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -0,0 +1,586 @@
+    if duration.Seconds() > 10 {
+        w.WriteHeader(500)
+        w.Write([]byte(fmt.Sprintf("error: %v", duration.Seconds())))
+    } else {
+        w.WriteHeader(200)
+        w.Write([]byte("ok"))
+    }
+})
+```
+
+O kubelet inicia a execução dos `health checks` 3 segunds após o contêiner iniciar.
+Assim o primeiro par de `health checks` terá sucesso. Mas após 10 segundos, os `health checks` vão falhar, e o kubelet encerrará e reiniciará o contêiner.
+
+Para testar a checagem de _liveness_ HTTP, crie um Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/probe/http-liveness.yaml
+```
+
+Após 10 segundos, veja os eventos do Pod para conferir que o _liveness probes_ falhou 
+e o contêiner foi reiniciado:
+
+```shell
+kubectl describe pod liveness-http
+```
+
+Em versões anteriores à v1.13 (incluindo a v1.13), se a variável de ambiente
+`http_proxy` (ou `HTTP_PROXY`) estiver atribuída no nó em que o Pod estiver executando,
+o _liveness probe_ HTTP usará esse proxy.
+Em versões após a v1.13, configurações de variáveis de ambiente HTTP proxy locais 
+não afetam o _liveness probe_ HTTP. 
+
+## Defina o _liveness probe_ TCP
+
+O terceiro tipo de _liveness probe_ usa o socket TCP. Com essa configuração, 
+o kubelet tentará abrir um socket para o seu contêiner na porta especificada.
+Se ele conseguir estabelecer uma conexão, o contêiner é considerado íntegro, 
+se não conseguir é considerado como falho.
+
+{{< codenew file="pods/probe/tcp-liveness-readiness.yaml" >}}
+
+Como você pode ver, a configuração para uma verificação TCP é bastante semelhante 
+a uma verificação HTTP.
+Este exemplo usa ambos os testes, _readiness_ e _liveness_. O kubelet enviará o
+primeiro teste de _readiness_ 5 segundos após o início do contêiner. Dessa forma tentará
+conectar-se ao contêiner `GoProxy` na porta 8080. Se o teste for bem sucedido, o Pod
+será marcado como pronto. O Kubelet continuará executando essa verificação a cada 10
+segundos.
+
+Conjuntamente com o teste de _readiness_, essa configuração inclui um teste de _liveness_.
+O Kubelet executará a primeira verificação de _liveness_ 15 segundos após o contêiner
+iniciar. Semelhante à verificação de _readiness_, ele tentará se conectar ao contêiner
+`GoProxy` na porta 8080. Se a verificação de _liveness_ falhar, o contêiner
+será reiniciado.
+Para testar a verificação de _liveness_ TCP, crie um Pod:
+
+```shell
+kubectl apply -f https://k8s.io/examples/pods/probe/tcp-liveness-readiness.yaml
+```
+
+Após 15 segundos, veja os eventos do Pod para verificar os _liveness probes_:
+
+```shell
+kubectl describe pod goproxy
+```
+
+## Defina uma _liveness probe_ gRPC
+
+{{< feature-state for_k8s_version="v1.24" state="beta" >}}
+
+Se a sua aplicação implementa o [Protocolo de verificação de integridade gRPC](https://github.com/grpc/grpc/blob/master/doc/health-checking.md),
+este exemplo mostra como configurar o kubernetes para usar usá-la para a verificação de _liveness_ da aplicação.
```

Comment:
"Kubernetes" instead of "kubernetes". Check all of it above.

typo: "usar usá-la"